lib/mix/lib/mix/tasks/deps.compile.ex

```diff
@@ -189,7 +189,10 @@ defmodule Mix.Tasks.Deps.Compile do
     lib_path = Path.join(config[:env_path], "lib/*/ebin")
 
     env = [{"REBAR_CONFIG", config_path}, {"TERM", "dumb"}]
```

Comment:
Instead of doing it with `System.put_env`, you can define the environemnt variable here. :)

Reply:
🤣 Got it!